app-k9mail/src/main/kotlin/app/k9mail/K9KoinModule.kt

```diff
@@ -3,9 +3,11 @@ package app.k9mail
 import app.k9mail.auth.K9OAuthConfigurationFactory
 import app.k9mail.core.common.oauth.OAuthConfigurationFactory
 import app.k9mail.core.common.provider.AppNameProvider
+import app.k9mail.core.featureflag.FeatureFlagFactory
 import app.k9mail.dev.developmentModuleAdditions
 import app.k9mail.feature.launcher.FeatureLauncherExternalContract.FeatureThemeProvider
 import app.k9mail.feature.widget.shortcut.LauncherShortcutActivity
+import app.k9mail.featureflag.K9FeatureFlagFactory
```

Comment:
Would it make sense to move these into `app.k9mail.feature.K9FeatureFlagFactory` and `net.thunderbird.android.feature.TbFeatureFlagFactory` ? I'm not sure a completely separate namespace for featureflags are needed, unless creating the actual feature flags would require new files?